src/app/donation-start/donation-start.component.ts

```diff
@@ -673,7 +675,7 @@ export class DonationStartComponent implements AfterContentChecked, AfterContent
     // Else settlement is via a new or saved card (including wallets / Payment Request Buttons).
     const result = this.paymentGroup.value.useSavedCard
         ? await this.stripeService.confirmPaymentWithSavedMethod(this.donation, this.stripeFirstSavedMethod as PaymentMethod)
-        : await this.stripeService.confirmPaymentWithNewCardOrPRB(this.donation, this.card);
+        : await this.stripeService.confirmPaymentWithNewCardOrPRB(this.donation, this.card as StripeCardElement);
```

Comment:
@bdsl I think it should be a reasonably safe assumption since we try to mount the Stripe.js element unconditionally and this should happen well after the page load, but I do agree we should avoid these where possible. The type warning (without `as`) seemed to be new for me, and this seemed like a quick win for removing it and clarifying the expected type in the code.